src/test/java/org/kiwiproject/dropwizard/error/ErrorContextUtilitiesTest.java

```diff
@@ -176,4 +184,48 @@ void shouldSkipRegisteringHealthCheck() {
             verifyNoInteractions(healthChecks);
         }
     }
+
+    @Nested
+    class RegisterCleanupJobOrNull {
+        private ApplicationErrorDao errorDao;
+        private CleanupConfig config;
+        private LifecycleEnvironment lifecycleEnvironment;
+
+        @BeforeEach
+        void setUp() {
+            errorDao = mock(ApplicationErrorDao.class);
+            config = new CleanupConfig();
+            lifecycleEnvironment = environment.lifecycle();
+        }
+
+        @Test
+        void shouldRegisterCleanupJob() {
+            var executor = mock(ScheduledExecutorService.class);
+            var executorBuilder = mock(ScheduledExecutorServiceBuilder.class);
+            when(executorBuilder.build()).thenReturn(executor);
+            when(lifecycleEnvironment.scheduledExecutorService(config.getCleanupJobName(), true))
```

Comment:
Be generic here using `anyString()` and `anyBoolean()` since you do the verification below.